src/_Base.sass

```diff
@@ -22,3 +22,8 @@ body
 	font-weight: 300
 	letter-spacing: .01em
 	line-height: 1.6
+
+// normalize.css sets font-family to sans-serif. The default sans-serif font
+// for Mac is Helvetica, but it has a strange baseline so the text is not vertically centered.
+button, input, optgroup, select, textarea
+	font-family: 'Roboto', 'Helvetica Neue', 'Arial', sans-serif
```

Comment:
@ndbroadbent why not define [here: _Base.sass](https://github.com/milligram/milligram/blob/master/src/_Base.sass#L20)